pkg/operator/ceph/object/config.go

```diff
@@ -40,6 +40,10 @@ caps osd = "allow rwx"
 	certKeyName               = "cert"
 	certFilename              = "rgw-cert.pem"
 	rgwPortInternalPort int32 = 8080
+	vaultVolumeName           = "rook-ceph-rgw-vault"
+	vaultDir                  = "/etc/ceph/vault"
+	vaultFilename             = "vault.token"
+	vaultKeyName              = "token"
```

Comment:
I believe we have all of those already in another package. Do not duplicate constants.